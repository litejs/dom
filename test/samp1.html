<!DOCTYPE html>
<html lang=en data-a="http://www.w3.org/1999/xhtml" data-b="http://opengraphprotocol.org/schema/"><head>
<title>1 + 2 = 3</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<style type="text/css">
/*<![CDATA[*/
body { background-image: url("marble.png?width=300&height=300") }
/*]]>*/
</style>
<link rel="stylesheet" type="text/css" href="styles/a.css?123">
<!--[if lt IE 8]><link rel="stylesheet" type="text/css" href="styles/ie.css?123"><![endif]-->
<script type="text/javascript" async="" src="http://www.google-analytics.com/ga.js"></script>
<script type="text/javascript" >
//<![CDATA[
document.write("<");
//]]>
</script>
<script type="module" src="main.js"></script> <script nomodule src="fallback.js"></script>
<style type="text/css">.class{position:absolute;}</style></head>
<body onload="
	alert('&quot;code&quot;');
" class="reset page"><!-- table --!>
    <ul class="gr">
            <li class="home"><span title="&amp;&lt;&gt;&apos;&quot;">&amp;&lt;&gt;&apos;&quot;</span ><img src="home.gif" alt="Home"></li>
    </ul>
        <div id="header" class="tpl-header"><!--My favorite operators are > and <!-->
	<iframe style="display:none" tabindex="-1"></iframe>
	</div>
</body>
</html>
</invalid>
