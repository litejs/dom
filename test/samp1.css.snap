@import url(fineprint.css) print;
@import url(bluish.css) print,screen;
@import'common.css'screen;
@import url(landscape.css) screen and (orientation:landscape);
@namespace url(http://www.w3.org/1999/xhtml);
@namespace svg url(http://www.w3.org/2000/svg);
@media (min-width:500px){
body{color:blue}
}
@font-face{font-family:'Trickster';src:local('Trickster'),url(trickster-COLRv1.otf) format('opentype') tech(color-COLRv1),url(trickster-outline.otf) format('opentype'),url(trickster-outline.woff) format('woff')}
@counter-style thumbs{system:cyclic;symbols:'\1F44D';suffix:' '}
ul{list-style:thumbs}
a{top:1px}
svg|a{top:2px}
*|a{top:3px}
@page{margin:1cm}
@keyframes slidein{
from{transform:translateX(0%)}
to{transform:translateX(100%)}
}